docs/running_on_rover.md

```diff
@@ -0,0 +1,19 @@
+# Running on Rover
+
+Instructions on running code on Viator's computer
+
+1. Setup router and connect to the wifi network `trickfirerouter` on your computer (the router's password is on the router itself).
+  - To get wifi to access Github, connect the ethernet cable to the router and the ethernet plug that is located overhead above the tables in the middle.
+2. Turn the rover computer on by plugging in the C-port charger. You will know the computer is on if the LED lights up.
+3. Run the commands below in Powershell if on windows or in your OS's terminal. When prompted for a password, please ask a software lead for it.
+
+```
+> ssh trickfire@192.168.0.145
+In Bash Now
```

Comment:
also you can mention that the password to both the router and orin is just `trickfire`

Reply:
I think this is a question I want to bring up at the next software meeting regarding security vs how accessible we want to make the rover, as I am scared in the future that there could be an accident.

Reply:
i mentioned this to vlad, and we both agreed this would not be an issue, but having a full discussion wouldn't be an issue. i unfortunately cant host a meeting this week however, as i am quite busy

Reply:
All good. I just wanted to verify both software leads are aware and that it is already on the internet.